x/evm/types/config.go

```diff
@@ -2,16 +2,18 @@ package types
 
 // EVMConfig contains all EVM module configuration values
 type EVMConfig struct {
-	Name       string `mapstructure:"name"`
-	RPCAddr    string `mapstructure:"rpc_addr"`
-	WithBridge bool   `mapstructure:"start-with-bridge"`
+	Name             string `mapstructure:"name"`
+	RPCAddr          string `mapstructure:"rpc_addr"`
+	WithBridge       bool   `mapstructure:"start-with-bridge"`
+	RPCModuleSupport bool   `mapstructure:"rpc-module-support"`
```

Comment:
This name is quite weird. It's basically saying if it's set to `true`, then do not try to detect whether or not it's beyond the standard ethereum jsonrpc. Maybe something like `enable-rpc-detection`.